### ISSUE TITLE
Unable to create application / modify permissions - version 1.17..0

### ISSUE THREAD

**Issue by raghunandanbs** [spinnaker/spinnaker#5113]
### Issue Summary:
Unable to create application or modify application permissions in spinnaker version 1.17.0

### Cloud Provider(s):
AWS, GCP

### Environment:
Local Debian / Distributed

### Description:
Following is the logs that I can see in Orca :
```
2019-11-07 13:24:45.331  INFO 802 --- [io-8080-exec-10] c.n.spinnaker.fiat.shared.FiatService    : ---> HTTP GET http://fiat.internal/authorize/<username1>
2019-11-07 13:24:45.335  INFO 802 --- [io-8080-exec-10] c.n.spinnaker.fiat.shared.FiatService    : <--- HTTP 200 http://fiat.internal/authorize/<username2> (4ms)
2019-11-07 13:24:47.436  WARN 802 --- [nio-8080-exec-4] c.n.s.f.shared.FiatPermissionEvaluator   : determine whether <username> can create resource com.netflix.spinnaker.front50.model.application.Application(TESTTEST, null, <email>, null, null, null, , null, null, null, null, null, null, null, testtest, [], [name:testtest, description:null,                                                                                                                                                   email:<email>, updateTs:null, createTs:null, details:[instancePort:80, chaosMonkey:[meanTimeBetweenKills                                                                                                                                                  InWorkDays:2, regionsAreIndependent:true, minTimeBetweenKillsInWorkDays:1, grouping:cluster, enabled:false, exceptions:[]                                                                                                                                                  ], providerSettings:[gce:[associatePublicIpAddress:false], aws:[useAmiBlockDeviceMappings:false]], user:<username>], c                                                                                                                                                  loudProviders:]) failed. Retrying in 3795ms

java.lang.IllegalArgumentException: FiatService.canCreate: Must have either a return type or Callback as last argument.
        at retrofit.RestMethodInfo.methodError(RestMethodInfo.java:107) ~[retrofit-1.9.0.jar:na]
        at retrofit.RestMethodInfo.parseResponseType(RestMethodInfo.java:270) ~[retrofit-1.9.0.jar:na]
        at retrofit.RestMethodInfo.<init>(RestMethodInfo.java:97) ~[retrofit-1.9.0.jar:na]
        at retrofit.RestAdapter.getMethodInfo(RestAdapter.java:213) ~[retrofit-1.9.0.jar:na]
        at retrofit.RestAdapter$RestHandler.invoke(RestAdapter.java:236) ~[retrofit-1.9.0.jar:na]
        at com.sun.proxy.$Proxy142.canCreate(Unknown Source) ~[na:na]
        at com.netflix.spinnaker.fiat.shared.FiatPermissionEvaluator.lambda$null$0(FiatPermissionEvaluator.java:166) [fia                                                                                                                                                  t-api-1.8.2.jar:1.8.2]
        at com.netflix.spinnaker.fiat.shared.FiatPermissionEvaluator$ExponentialBackoffRetryHandler.retry(FiatPermissionE                                                                                                                                                  valuator.java:97) ~[fiat-api-1.8.2.jar:1.8.2]
        at com.netflix.spinnaker.fiat.shared.FiatPermissionEvaluator.lambda$canCreate$1(FiatPermissionEvaluator.java:162)                                                                                                                                                   [fiat-api-1.8.2.jar:1.8.2]
        at com.netflix.spinnaker.security.AuthenticatedRequest.lambda$propagate$0(AuthenticatedRequest.java:127) ~[kork-s                                                                                                                                                  ecurity-6.15.1.jar:6.15.1]
        at com.netflix.spinnaker.fiat.shared.FiatPermissionEvaluator.canCreate(FiatPermissionEvaluator.java:184) [fiat-ap                                                                                                                                                  i-1.8.2.jar:1.8.2]
```

**Comment by ealebed** [comment]
The same issue as https://github.com/spinnaker/spinnaker/issues/5109

**Comment by maggieneterval** [comment]
Hi @raghunandanbs  and @ealebed! Going to close this issue because, as @ealebed mentioned, it is a duplicate of https://github.com/spinnaker/spinnaker/issues/5109.

The fix will be released with 1.17.1 on Monday. Thanks!
